subworkflows/local/bam_variant_calling_germline_manta/main.nf

```diff
@@ -10,57 +7,30 @@ workflow BAM_VARIANT_CALLING_GERMLINE_MANTA {
     dict          // channel: [optional]  [ meta, dict ]
     fasta         // channel: [mandatory] [ fasta ]
     fasta_fai     // channel: [mandatory] [ fasta_fai ]
-    intervals     // channel: [mandatory] [ interval.bed.gz, interval.bed.gz.tbi, num_intervals ] or [ [], [], 0 ] if no intervals
+    intervals     // channel: [mandatory] [ interval.bed.gz, interval.bed.gz.tbi] or [ [], []] if no intervals; intervals file contains all intervals
 
     main:
     versions = Channel.empty()
 
-    // Combine cram and intervals for spread and gather strategy
-    cram_intervals = cram.combine(intervals)
-        // Move num_intervals to meta map
-        .map{ meta, cram, crai, intervals, intervals_index, num_intervals -> [ meta + [ num_intervals:num_intervals ], cram, crai, intervals, intervals_index ] }
+    // Combine cram and intervals, account for 0 intervals
+    cram_intervals = cram.combine(intervals).map{ it ->
```

Comment:
Can you use `meta, cram, crai, intervals, intervals_index, num_intervals` instead of `it` to be more explicit about what is happening there?

Reply:
no, because when intervals is empty there are no elements in the channel and it just looks like [meta,cram,crai] after combining.

Also we don't have num_intervals anymore here, since it is only one bed file

Reply:
oooh I see